server/app.js

```diff
@@ -16,6 +16,10 @@ app.use(bodyParser.urlencoded({ extended: true }));
 
 const port = process.env.PORT || 6000;
 
+app.get('/', (req, res) => res.status(200).json({
+  message: 'Welcome to Politico',
```

Comment:
Please check the project document for the correct response message format.

Reply:
Acknowledged with thanks.
I have implemented the changes based on the correct message format